concepticondata/conceptlists/Bowern-2021-203c.tsv

```diff
@@ -0,0 +1,204 @@
+Bowern-2021-203c-38	38	clapper 	705	acculturation	Culture Vocabulary		
+Bowern-2021-203c-39	39	clapsticks 	773	culture-mythology	Culture Vocabulary		
+Bowern-2021-203c-40	40	clothing 	676	culture-mythology	Culture Vocabulary	1895	CLOTHES
+Bowern-2021-203c-41	41	club 	621	subsistence tool	Culture Vocabulary	1763	CLUB
+Bowern-2021-203c-42	42	club (nulla-nulla) 	774	subsistence tool	Culture Vocabulary		
+Bowern-2021-203c-43	43	coca 	153	flora-fauna	Culture Vocabulary	137	COCA
+Bowern-2021-203c-44	44	coca 	717	narcotics	Culture Vocabulary	137	COCA
+Bowern-2021-203c-45	45	cook food 	623	food	Culture Vocabulary	1100	COOK (SOMETHING)
+Bowern-2021-203c-46	46	cook in earth oven 	775	food	Culture Vocabulary		
+Bowern-2021-203c-47	47	corn 	700	food	Culture Vocabulary	605	GRAIN
+Bowern-2021-203c-48	48	corpse 	624	food	Culture Vocabulary		
+Bowern-2021-203c-49	49	cradleboard 	697	culture-material	Culture Vocabulary		
+Bowern-2021-203c-50	50	cremate 	689	culture-mythology	Culture Vocabulary		
+Bowern-2021-203c-51	51	Curupira (spirit type) 	718	culture-mythology	Culture Vocabulary		
+Bowern-2021-203c-52	52	damper 	776	food	Culture Vocabulary		
+Bowern-2021-203c-53	53	dance (generic) 	625	culture-mythology	Culture Vocabulary	3472	DANCE (MOVEMENT)
+Bowern-2021-203c-54	54	dart (blowgun) 	719	subsistence tool	Culture Vocabulary	2587	DARTS
+Bowern-2021-203c-55	55	Datura wrightii 	653	narcotics	Culture Vocabulary		
+Bowern-2021-203c-56	56	deity/powerful spirit/culture figure 	720	culture-mythology	Culture Vocabulary	3231	DEITY
+Bowern-2021-203c-57	57	digging stick 	627	subsistence tool	Culture Vocabulary	339	DIGGING STICK
+Bowern-2021-203c-58	58	dipper 	628	subsistence tool	Culture Vocabulary		
+Bowern-2021-203c-59	59	dipper (small coolamon) 	777	subsistence tool	Culture Vocabulary		
+Bowern-2021-203c-60	60	dream 	629	culture-mythology	Culture Vocabulary	1920	DREAM (SOMETHING)
+Bowern-2021-203c-61	61	dream	397	mental	Culture Vocabulary	1920	DREAM (SOMETHING)
+Bowern-2021-203c-62	62	dreaming 	778	culture-mythology	Culture Vocabulary	1920	DREAM (SOMETHING)
+Bowern-2021-203c-63	63	drone pipe 	630	culture-mythology	Culture Vocabulary		
+Bowern-2021-203c-64	64	drum 	631	culture-mythology	Culture Vocabulary	908	DRUM
+Bowern-2021-203c-65	65	drum (large signal) 	829	culture	Culture Vocabulary		
+Bowern-2021-203c-66	66	earring 	633	other	Culture Vocabulary	770	EARRING
+Bowern-2021-203c-67	67	earth oven 	818	food	Culture Vocabulary		
+Bowern-2021-203c-68	68	fan 	721	subsistence tool	Culture Vocabulary	1971	FAN (OBJECT)
+Bowern-2021-203c-69	69	feather headband 	604	dress	Culture Vocabulary		
+Bowern-2021-203c-70	70	feather headdress 	722	culture-mythology	Culture Vocabulary		
+Bowern-2021-203c-71	71	fence, palisade 	742	other	Culture Vocabulary	1690	FENCE
+Bowern-2021-203c-72	72	fermented drink 	830	culture	Culture Vocabulary		
+Bowern-2021-203c-73	73	fetish, charm 	723	culture-mythology	Culture Vocabulary	2804	AMULET
```

Comment:
https://huntergatherer.la.utexas.edu/lexical/feature/723 seems more specific than AMULET. I'd unlink this.